packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx

```diff
@@ -171,7 +214,19 @@ const PostsPagePostHeader = ({post, classes}: {
         <PostsVote post={post} />
       </div>}
     </div>
-    
+    {podcastEpisode && <div className={classNames({ [classes.hideEmbeddedPlayer]: !showEmbeddedPlayer })}>
```

Comment:
I'm open to extracting this to its own component.

Reply:
Yeah I think this is worth proactively componentizing (I wouldn't block the PR on it, but think it'd be good to do soon).